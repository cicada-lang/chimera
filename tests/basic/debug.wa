find [x, y] debug 3 {
  x = true
  y = false
}
