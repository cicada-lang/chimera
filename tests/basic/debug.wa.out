---
step_count: 0
queues:
  - solution: {}
    goals:
      - '"?x" = true'
      - '"?y" = false'
query_pattern: (x, y)
solutions: []

---
step_count: 1
queues:
  - solution:
      x: true
    goals:
      - '"?y" = false'
query_pattern: (x, y)
solutions: []

---
step_count: 2
queues:
  - solution:
      y: false
      x: true
    goals: []
query_pattern: (x, y)
solutions: []

[ [true, false] ]
